lib/diagnostic.rb

```diff
@@ -102,18 +116,24 @@ def eat_meat(food)
       puts 'Yuck!'
     end
   end
-end
 
 ##
-# your response here
+class Lion
+  include Carnivorous
+  def roar
+    p 'ROAR!'
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+Mixins are used when the class including it shares only a minor attribute.
+For example people and computer both sleep but otherwise they share no other
+simlarities.
```

Comment:
In direct inheritance, you're limited to inheriting from a single source. In composition, you can draw from as many modules as you like.